R/constants.R

```diff
@@ -130,9 +130,11 @@ ELECTRICITY_INPUT_FUELS<- c( "biomass", "coal", "gas", "refined liquids" )
 # and should contain all historical years used by other modules
 socioeconomics.MADDISON_HISTORICAL_YEARS <- seq(1700, 1900, 50) # Years for which to use Maddison data
 socioeconomics.UN_HISTORICAL_YEARS <- c(1950, 1971:2010) # Years for which to use UN data
```

Comment:
I'm not sure how that would work in this case. The SSP values begin in 2010 and, since we don't want a jump in the population values when we move from the UN to the SSP data, the GCAM SSP population values are not exactly the population values in the SSP database, but the ratio of the future SSP values to the 2010 SSP value times the UN 2010 population. If the "future" years started in 2001, how would we estimate the SSP population projections?
I don't know that we can't, I'm just not sure how we would. Also, possibly we wouldn't want to. Since population is always exogenous, the pre-2015 values are all always going to be the same (we're not projecting 2020 based on 2015, that's determined by the scenario, and the historic years are all fixed--they're not something that would change depending on the definition of "historical" and "future" years in the model.
Not sure if I answered your question, though.